app.js

```diff
@@ -1,25 +1,51 @@
 // IMPORT PACKAGES
 // Here you should import the required packages for your Express app: `express` and `morgan`
+const express = require("express")
 
-
-
+const morgan = require("morgan")
 // CREATE EXPRESS APP
 // Here you should create your Express app:
+const app = express()
 
+app.use(morgan('dev'))
 
 
+app.use(express.static('public'))
+
+app.use(express.json())
 // MIDDLEWARE
 // Here you should set up the required middleware:
 // - `express.static()` to serve static files from the `public` folder
 // - `express.json()` to parse incoming requests with JSON payloads
 // - `morgan` logger to log all incoming requests
-
-
+console.log(__dirname )
 
 // ROUTES
 // Start defining your routes here:
-
-
+app.get('/',(req,res)=>{
+ res.sendFile(__dirname + '/views/home.html')
+
+})
+app.get('/blog',(req,res)=>{
+    res.sendFile(__dirname + '/views/blog.html')
+   
+   })
+   app.get('/api/projects',(req,res)=>{
+    res.sendFile(__dirname + '/data/projects.json')
+   
+   })  
+   
+   app.get('/api/articles',(req,res)=>{
+    res.sendFile(__dirname + '/data/articles.json')
```

Comment:
And in this case you will send something like this 
```suggestion
   res.json(articles)
```